reference-implementation/lib/readable-byte-stream.js

```diff
@@ -0,0 +1,965 @@
+const assert = require('assert');
+import { CreateIterResultObject, InvokeOrNoop, PromiseInvokeOrNoop, typeIsObject } from './helpers';
+
+export default class ReadableByteStream {
+  constructor(underlyingByteSource = {}) {
+    this._state = 'readable';
+    this._reader = undefined;
+    this._storedError = undefined;
+
+    this._controller = undefined;
```

Comment:
AAh! This is required. Reverted with a comment. a1745016eae6ec0dd8e49c938e43c89744d37499